chore: add go sdk storage examples (#1081)
momentohq/public-dev-docs: docs/sdks/go/storage.mdx

# Getting Started with Momento Storage in Go

If you need to get going quickly with Go and Momento Storage, this page contains the basic API calls you'll need. [Check the Go SDK examples](https://github.com/momentohq/client-sdk-go/tree/main/examples) for complete, working examples including build configuration files.

## Install the Momento SDK

The Momento SDK is available on Go Packages: [client-sdk-go](https://pkg.go.dev/github.com/momentohq/client-sdk-go).

:::tip
Visit [Go Packages](https://pkg.go.dev/github.com/momentohq/client-sdk-go) to find the latest available version of the SDK.
:::

Install the client library in an existing Go project:

```bash
go get github.com/momentohq/client-sdk-go
```

## Set up your API key

You'll need a Momento API key to authenticate with Momento.  You can get one from the [Momento Web Console](https://console.gomomento.com/api-keys).
Once you have your API key, store it in an environment variable so that the Momento client can consume it:

```
export MOMENTO_API_KEY=<your Momento API key here>
```

## Import libraries and connect to return a StorageClient object

> This code sets up the main function, the necessary imports, and the StorageClient instantiation that each of the other functions will need to call.

Reviewer: ```suggestion
This code sets up the main function, the necessary imports, and the StorageClient that will be used to interact with your store.
```
